eppy/tests/test_function_helpers.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pytest
+from six import StringIO
+
+import eppy.function_helpers as fh
+from eppy.bunch_subclass import register_epbunch_function
+from eppy.iddcurrent import iddcurrent
+from eppy.modeleditor import IDF
+from eppy.tests.test_bunch_subclass import idftxt
+
+
+class TestRegisterFunction:
+    @pytest.fixture()
+    def IDF(self):
+        iddfhandle = StringIO(iddcurrent.iddtxt)
+
+        if IDF.getiddname() == None:
+            IDF.setiddname(iddfhandle)
+        yield IDF
+
+    @pytest.fixture()
+    def idf(self, IDF):
+        idfhandle = StringIO(idftxt)
+        yield IDF(idfhandle)
+
+    def test_register_function_area(self, idf):
+        """Add a function that comes default with eppy"""
+
+        @register_epbunch_function(
+            "new_area",
+            keys=[
+                "BuildingSurface:Detailed",
+                "Wall:Detailed",
+                "RoofCeiling:Detailed",
+                "Floor:Detailed",
+                "FenestrationSurface:Detailed",
+                "Shading:Site:Detailed",
+                "Shading:Building:Detailed",
+                "Shading:Zone:Detailed",
+            ],
+        )
+        @property
+        def area(abunch):
+            return fh.area(abunch)
+
+        surf = idf.idfobjects["BuildingSurface:Detailed".upper()][0]
+        assert surf.area
```

Comment:
Shouldn't this be testing for `surf.new_area`?

Reply:
handled in 3080f0d607266f6bae0554640f89b0740071b1aa